app/screens/setting/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Text, View, Platform, Modal, Pressable, Alert } from "react-native";
+import { useRootStore } from "../../store/rootStore";
+import styles from "../../styles";
+import EthSetting from "./EthSetting";
+
+const Setting = () => {
+  const rootStore = useRootStore();
+  const settings = rootStore.Assets.selectedSetting;
+
+  if (!settings) {
+    return <View>{/* TODO: error handling for this case */}</View>;
+  }
+
+  const renderSetting = (key: string) => {
+    switch (key) {
+      case "1x00":
```

Comment:
why do the settings need a key? I still havent quite got my head around what the key is for. Is the the key the identiy from DID? @bitcoinbrisbane